src/main/features/ccj/views/pay-by-set-date.njk

```diff
@@ -15,7 +15,7 @@
           {{ t('When do you want them to pay?') }}
         </h1>
 
-        {{ dateInput(name = 'date', form = form, hint = t('For example, {{ futureDate }}', { futureDate: futureDate})) }}
+        {{ dateInput(name = 'date', form = form, hint = t('For example, {{ futureDate }}', { futureDate: futureDate.format('DD MM YYYY') })) }}
```

Comment:
Please use filter `date` unless this is the only place in our app where we need different format of date displayed.

```
{ futureDate: futureDate | date }
```

Example:
`src/main/features/ccj/views/confirmation.njk` line 17